### PR TITLE
Change script path in map dependancies to match with LEL #6978

### LINKED ISSUE #6978: Change script path in map dependancies to match with LEL
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file _mapDependencies.html.erb in the `plots2` repository . 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Remove the line here:

https://github.com/publiclab/plots2/blob/3f239088f6028b2e87668f7cd9076295dfe4e79d/app/views/map/_mapDependencies.html.erb#L20-L21

and add:
```
<%= javascript_include_tag('/lib/leaflet-environmental-layers/lib/glify.js') %>
<%= javascript_include_tag('/lib/leaflet-environmental-layers/lib/leaflet-fullUrlHash.js') %>
```

to match what we currently have on the [leaflet environmental layers](https://github.com/publiclab/leaflet-environmental-layers) library.

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by bearkruz**
[Fixes #6978 
](https://github.com/publiclab/plots2/issues/6978)

The noted error in the log is:
"ActionController::RoutingError: ActionController::RoutingError: No route matches [GET] "/lib/leaflet-environmental-layers/lib/glify.js"

This is likely because the LEL package in plots2 is not updated with the changes from the LEL repo. The folders were moved just recently.

### CODE REVIEW

File: app/views/map/_mapDependencies.html.erb [hunk 1]
```diff
@@ -18,4 +18,4 @@
 <%= javascript_include_tag('https://cdnjs.cloudflare.com/ajax/libs/OverlappingMarkerSpiderfier-Leaflet/0.2.6/oms.min.js') %>
 
 <%= javascript_include_tag('/lib/leaflet-environmental-layers/example/lib/glify.js') %>
```
Comment (pythongiant) [inline app/views/map/_mapDependencies.html.erb]: Hi @bearkruz,
You were supposed to change it to `<%= javascript_include_tag('/lib/leaflet-environmental-layers/lib/glify.js') %>` as mentioned in the issue. 
Thanks

File: app/views/map/_mapDependencies.html.erb [hunk 2]
```diff
@@ -18,4 +18,4 @@
...
-
+<%= javascript_include_tag('/lib/leaflet-environmental-layers/example/lib/leaflet-fullUrlHash.js') %>
```
Comment (pythongiant) [inline app/views/map/_mapDependencies.html.erb]: Same here but change to `<%= javascript_include_tag('/lib/leaflet-environmental-layers/lib/leaflet-fullUrlHash.js') %>`

File: app/views/map/_mapDependencies.html.erb (hunk 1 above)
Comment (bearkruz) [reply app/views/map/_mapDependencies.html.erb]: Hi @pythongiant. in #7029, I did this and received the same error message. I updated to use 'example' in the link, as seen in the original line #20, to try overcome the error message. Whilst this left the same error message,  I've just updated the code in line with the task sample.

File: app/views/map/_mapDependencies.html.erb (hunk 2 above)
Comment (bearkruz) [reply app/views/map/_mapDependencies.html.erb]: Hi @pythongiant. in #7029, I did this and received the same error message. I updated to use 'example' in the link, as seen in the original line #20, to try overcome the error message. Whilst this left the same error message,  I've just updated the code in line with the task sample.

**Comment by cesswairimu** [comment]
Restarting travis

**Comment by crisner** [comment]
@bearkruz, I apologize for the delay in my response. I just came across your message on gitter. So the issue here is the LEL library needs to be published for the changes on it to reflect on plots2. 

We should put this on hold for now. I'll let you know once the changes are published and this is ready to work on. I should have followed up on this the last time I replied to your message. I did not have a good grasp of how versioning worked at that time. I am truly sorry for the inconvenience.

**Comment by jywarren** [comment]
Ah, i see tests are still not passing. Should we perhaps somehow combine this with #7147 and could that resolve tests for both? Are they interdependent? 

Thank you!!!

**Comment by jywarren** [comment]
Potentially yes... 

```
ERROR["test_adding_a_tag_to_a_user_profile", #<Minitest::Reporters::Suite:0x00007f5651d4da10 @name="TagTest">, 8.329939246000038]
 test_adding_a_tag_to_a_user_profile#TagTest (8.33s)
ActionController::RoutingError:         ActionController::RoutingError: No route matches [GET] "/lib/leaflet-environmental-layers/lib/leaflet-fullUrlHash.js"
```

**Comment by crisner** [comment]
Yes. The tests here can't find the new route as it doesn't exist in the version of LEL used here and the tests in #7147 still refer to the old route in map dependancies.

**Comment by jywarren** [comment]
OK! @bearkruz did you want to try adding the changes from https://github.com/publiclab/plots2/pull/7147/files to get this to pass? 

Alternatively @crisner if you'd like to move this forward, you could open a PR that pulls in both these PRs commits, and I can merge them together through your PR. Thanks!

**Comment by crisner** [comment]
@bearkruz, I had pulled in your commit from this PR and it has been resolved in #7157. Thanks for working on this.

**Comment by bearkruz** [comment]
@crisner Fantastic - thanks for that update! 🚀

**Comment by SidharthBansal** [comment]
Kindly resolve conflicts

**Comment by crisner** [comment]
@SidharthBansal, a heads up. As suggested in https://github.com/publiclab/plots2/pull/7030#issuecomment-572197799, @bearkruz's commits from this PR were pulled in and the issue was resolved in #7157. I think this can be closed as @bearkruz's contribution was pulled in but I wasn't sure if that's the way to proceed.  Thanks.

**Comment by SidharthBansal** [comment]
So we can close this. We will reopen it if needed

**Pull request closed without merging**